op-batcher/batcher/config.go

```diff
@@ -88,17 +88,14 @@ type CLIConfig struct {
 	Stopped bool
 
 	TxMgrConfig      txmgr.CLIConfig
-	RPCConfig        rpc.CLIConfig
 	LogConfig        oplog.CLIConfig
 	MetricsConfig    opmetrics.CLIConfig
 	PprofConfig      oppprof.CLIConfig
 	CompressorConfig compressor.CLIConfig
+	RPCFlag          oprpc.CLIConfig
 }
 
 func (c CLIConfig) Check() error {
-	if err := c.RPCConfig.Check(); err != nil {
```

Comment:
Should still be checking the config

Reply:
Thanks for catching this. I've updated it. It looks like we haven't even been calling this function from the beginning.